verifier/src/error.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2023 The MobileCoin Foundation
+
+//! Errors that can occur during verification
+
+use mc_sgx_dcap_types::TcbError;
+
+pub type Result<T> = core::result::Result<T, Error>;
+
+/// Error working with quote evidence
+#[derive(displaydoc::Display, Debug)]
+pub enum Error {
+    /// Error converting from DER {0}
+    Der(der::Error),
```

Comment:
I contemplated placing the x509Error in here, but I think the x509 certificate logic is separate functionality, it just happens to live in this crate as a convenient place to host the functionality.